gabbi/simple_wsgi.py

```diff
@@ -51,6 +51,9 @@ def __call__(self, environ, start_response):
             ('X-Gabbi-url', request_url),
         ]
 
+        if request_method == 'DIE':
+            raise Exception('because you asked me to')
```

Comment:
No it was. There are various features in gabbi that are a bit hard to properly test from gabbi itself, so rather than actually test them the simple_wsgi app has ways to ask that certain pieces of the code are traversed. The line above combined with the test at line 109 in self.yaml insure that the exception handling is at least encountered.